### PR TITLE
🚛 return labels rest api company business unit connector

### PR DESCRIPTION
**Pull request by DerCalli**
**Changelog**

- module expands return label with company-business-unit email

### CODE REVIEW

File: bundles/return-labels-rest-api-company-business-unit-connector/LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2021 FOND OF GmbH
```
Comment (SpiderDan98) [inline bundles/return-labels-rest-api-company-business-unit-connector/LICENSE]: Update year

Comment (julianhyatt) [inline bundles/return-labels-rest-api-company-business-unit-connector/LICENSE]: 2022

File: bundles/return-labels-rest-api-company-business-unit-connector/composer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "fond-of-oryx/return-labels-rest-api-company-business-unit-connector",
+  "description": "Return Labels Rest Api Extension Module",
```
Comment (SpiderDan98) [inline bundles/return-labels-rest-api-company-business-unit-connector/composer.json:3]: Is this the correct description?

File: bundles/return-labels-rest-api-company-business-unit-connector/composer.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "spryker/code-sniffer": "^0.12"
+  },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "autoload": {
+    "psr-4": {
+      "FondOfOryx\\": "src/FondOfOryx/"
+    }
+  },
+  "autoload-dev": {
```
Comment (julianhyatt) [inline bundles/return-labels-rest-api-company-business-unit-connector/composer.json:25]: config is missing

**Review by julianhyatt — APPROVED** [review]

**Review by SpiderDan98 — APPROVED** [review]

**Pull request merged**